components/landlord/LandlordPage.tsx

```diff
@@ -36,6 +38,7 @@ const LandlordPage = ({ landlord, data }: IProps) => {
 	}
 
 	useEffect(() => {
+		console.log(data.otherLandlords)
```

Comment:
Remove Log

Reply:
Whoops lol I remember telling myself to delete that.